bob-backend/src/pearl/disk_controller/logger.rs

```diff
@@ -0,0 +1,55 @@
+use super::*;
+use chrono::Local;
+use tokio::fs::{File, OpenOptions};
+use tokio::io::AsyncWriteExt;
+
+#[derive(Clone, Debug)]
+pub(crate) struct DisksEventsLogger {
+    fd: Arc<RwLock<File>>,
+}
+
+impl DisksEventsLogger {
+    pub(crate) async fn new(filename: impl AsRef<Path>) -> Self {
+        let fd = if filename.as_ref().exists() {
```

Comment:
Probably new should return Result (or Maybe), so we avoid panics here (and can start server even if events can't be logged, simply printing errors)